test/unit/specs/Gambler.spec.js

```diff
@@ -0,0 +1,82 @@
+// var chai = require('chai');
+var sinon = require('sinon')
+import {store} from '../../../src/store/store'
```

Comment:
Absolute Import references like these make me nervous. If the directory structure gets changed in the future, this will break. 

Having said that though, I'm not sure the likelihood of directory structure changes in the future.